database/src/main/postgres/runs/V1.8.3__get_partitioning_checkpoint.sql

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- Function: runs.get_partitioning_checkpoints(JSONB, INT, TEXT)
+CREATE OR REPLACE FUNCTION runs.get_partitioning_checkpoints(
+    IN i_partitioning              JSONB,
+    IN i_limit                     INT DEFAULT 5,
+    IN i_checkpoint_name           TEXT DEFAULT NULL,
+    OUT status                     INTEGER,
+    OUT status_text                TEXT,
+    OUT id_checkpoint              UUID,
+    OUT checkpoint_name            TEXT,
+    OUT measure_name               TEXT,
+    OUT measure_columns            TEXT[],
+    OUT measurement_value          JSONB,
+    OUT checkpoint_start_time    TIMESTAMP WITH TIME ZONE,
+    OUT checkpoint_end_time        TIMESTAMP WITH TIME ZONE
+)
+    RETURNS SETOF record AS
+$$
+    -------------------------------------------------------------------------------
+--
+-- Function: runs.get_partitioning_checkpoints(JSONB, INT, TEXT)
+--      Returns all the checkpoint for the given partitioning and checkpoint name
+--
+-- Parameters:
+--      i_partitioning      - partitioning for requested checkpoints
+--      i_limit             - limit of the number of checkpoints to return
```

Comment:
Taking from `flows.get_flow_checkpoints`
```suggestion
--      i_limit             - limit of the number of checkpoints to return
--      i_limit                 - (optional) maximum number of checkpoints retrieved
--                                if 0 specified, all data will be returned, i.e. no limit will be applied
```